openapi.yaml

```diff
@@ -255,10 +240,8 @@ paths:
           required: false
           schema:
             type: string
-            enum:
-              - active
-              - suspended
-              - expired
+            allOf:
+             - $ref: '#/components/schemas/MemberStatusEnum'
```

Comment:
You can also use [yaml anchors](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) for these references. Just want to let you know and no need to change here :)

Reply:
sure :)